models/product.model.lkml

```diff
@@ -3,6 +3,23 @@ connection: "snowflake"
 # Limit include only to the ones really needed by this explore
 include: "/views/marts/product/*.view.lkml"
 
+explore: fct_nps_feedback {
+  label: "NPS Feedback"
+  group_label: "[New] NPS Feedback"
+
+  join: dim_cloud_customers {
+    relationship: one_to_one
```

Comment:
Since fact has multiple rows for the same server (different dates, different users etc), shouldn't this be many to one?

Reply:
updated to `many_to_one`